cloudbees-cd/es-to-os-migration/README.md

```diff
@@ -1 +1,98 @@
-Placeholder path for ES to OS README
+# Estimate data size of legacy CloudBees Analytics indexes
+
+In CloudBees CD/RO v2024.06.0, the CloudBees Analytics server was upgraded from Elasticsearch to OpenSearch. As part of the upgrade process from v2024.03.0 or earlier to v2024.06.0 or later, you must migrate your CloudBees Analytics to OpenSearch, as described on:
+
+* [Traditional migrations](https://docs.cloudbees.com/docs/cloudbees-cd/latest/troubleshooting/trad-upgrade-to-os)
+* [Kubernetes migrations](https://docs.cloudbees.com/docs/cloudbees-cd/latest/troubleshooting/k8s-upgrade-to-os)
+
+To help roughly estimate the size of legacy CloudBees Analytics indexes data that will be migrated to your updated CloudBees Analytics server,  CloudBees has provided the [reporting-data-reindex.pl](reporting-data-reindex.pl) utility.
+
+> **IMPORTANT**  
+> As described in the [known issues of the data migration](https://docs.cloudbees.com/docs/cloudbees-cd/latest/troubleshooting/data-migration-es-to-os#data-migration-known-issues), the automated processes CloudBees has supplied to automate data migration have a timeout of `180 minutes` per index. This is roughly equivalent to `32,000,000` documents, which may vary based on the complexity of individual documents (number of fields, etc.).   
+> 
+> If your indexes contain vastly more than 32,000,000 documents, or you reach the timeout limit for specific indexes, it may be necessary to split the specific index into a set of smaller indexes. For more information on this process, refer to the [Elasticsearch Split index API](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/indices-split-index.html) documentation.
```

Comment:
Unfortunately using the spit index API will not do what we need here. The spit index API spits the documents in the index into more shards by increasing the number of primary shards. But the total number of documents in the index stay the same. The fix we need is to split the documents into multiple indexes...let me think about this and get back to you if we can document some instructions for users on how to do this. For now, we can remove the line on split index API.